MFVNeutralino/python/AnalysisCuts_cfi.py

```diff
@@ -13,26 +17,31 @@
                                apply_presel = apply_presel,  # 1 = jets, 2 = el/mu, 3 = jets OR bjet/displaced dijet triggers, 4 = bjet/displaced dijet triggers veto HT trigger, 5 = MET trigger
                                                              # 6 = bjets/displaced dijet (HT agnostic)
                                require_met_filters = cms.bool(True) if use_MET_triggers else cms.bool(False),
-                               require_bquarks  = cms.bool(False),
-                               require_trigbit  = cms.bool(True),
+                               require_bquarks  = cms.bool(True) if use_btag_triggers else cms.bool(False),
```

Comment:
This is about gen-level bquarks rather than reco ones. I suspect this should just be false....? May even be a question for @shaunhogan